slides/src/content/introduction.md

```diff
@@ -37,4 +53,247 @@ FIXME
+1. Install Angular CLI -- `npm install --global @angular/cli`
+
+---
+
+## Creating a New Project
+
+`ng create [project-name]`
+
+This will:
+
+- Install Angular
+- Install node dependencies
+- Create scaffolding
+
+---
+
+## Project Structure
+
+- `package.json` -- project metadata
+- `node_modules` -- libraries and dependencies used by our application
+- `src` folder -- application code
+- `tsconfig.json` -- TypeScript configuration
+
+---
+
+## TypeScript
+
+- TypeScript is exactly like JavaScript, but with types
+- Has ES6 and ES7 features
+- Compiles to JavaScript before being run in the browser
+- No runtime support for types
+
+---
+
+## First Look at the Component
+
+```typescript
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: '<h1>{{title}}</h1>',
+  styles: ['h1 { color: font-size: 30px; }']
+})
+export class AppComponent {
+  title = 'app works!';
+}
+```
+
+- Import statement gives you access to Angular components
+- `@Component` is a decorator function that tells Angular this class is a component
+- `selector` is the way you intend to reference the component. Used like an HTML tag `<app-root></app-root>`
+- `template` is what gets rendered onto the page. For web applications, this is HTML
+- `styles` are css stylesheets that get applied to your component
+- `AppComponent` is a class that decides how the component should act
+
+---
+
+## What is a Component?
+
+- The core building block of an Angular application is the component
+- In Angular, a component is some chunk of application logic combined with something that gets displayed
+- Everything being displayed on any page should be associated with some component
+
+---
+
+## Component Template Syntax
+
+- `<h1>{{title}}</h1>` represents __interpolation__
+- Interpolation attempts to evaluate the expression between the curly braces and turn that into a string
+- Symbols (such as title) are evaluated within an expression context, in this case, the component's class members
+
+---
+
+## Generating a New Component
+
+- We'll use Angular CLI to create a new component
+- `ng generate component [component path]`
+- Component gets added to the list of declarations
+- Generated component follows the same structure as `AppComponent`
+- Prefix `app` on both components. This is an Angular best practice
+
+---
+
+## Integrating a Component
+
+Every component has a selector:
+
+*example.component.ts*
+```typescript
+@Component({
+	selector: 'app-example',
+	template: '<p>Multiple component example</p>'
+})
+export class ExampleComponent { }
+```
+
+---
+
+## Integrating a Component
+
+Use selector as an HTML tag within another component's template
+
+
+*app.component.ts*
+```typescript
+@Component({
+	selector: 'app-root',
+	template: `
+	  <h1>My Angular App</h1>
+	  <app-example></app-example>
+	`
+})
+export class AppComponent { }
+```
+[View Example](https://plnkr.co/edit/CMGx2TLUdPcL0D4tvGI9?p=preview)
```

Comment:
For engagements I've done, we typically expand the example link in the cases we can. I don't know if it makes sense to include a separate slide with a screenshot in that case.